docs/orchestrating-elastic-stack-applications/elasticsearch/orchestration.asciidoc

```diff
@@ -173,3 +173,68 @@ Specifically, if the Elasticsearch index settings demand a higher number of shar
 
 ** Adjust the Elasticsearch link:https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-update-settings.html[index settings] to a number of replicas that allow the desired node removal.
 ** Use link:https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#dynamic-index-settings[`auto_expand_replicas`] to automatically adjust the replicas to the number of data nodes in the cluster.
+
+[id="{p}-advanced-upgrade-control"]
+== Advanced control during rolling upgrades
+
+The rules (otherwise known as predicates) that the ECK operator follows during an Elasticsearch upgrade can be selectively disabled for certain scenarios where the ECK operator will not proceed with an Elasticsearch cluster upgrade because it deems it to be "unsafe".
+
+WARNING: Selectively disabling the predicates below are extremely risky, and carry a high chance of either data loss, or causing a cluster to become completely unavailable.  Use only if you are absolutely positive you are not causing permanent damage to an Elasticsearch cluster.
+
+* The following named predicates control the upgrade process
```

Comment:
Maybe also mention that all the predicates can be disabled using `"*"`

Reply:
good point.